README.md

```diff
@@ -68,16 +111,217 @@ you can also chain components names `react-component <ComponentName> <ComponentN
+
+    react-component init -t
+
+This will initialize local settings and local templating.
+
+The doT templating language uses `{{}}` for inertoplation and javascript execution. Here are some basic uses of doT:
+
+<!-- table>(thead>tr>th*3)+tbody>tr*3>td*3 -->
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Code</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Interpolation</td>
+            <td>{{= someVariable }}</td>
+            <td>Replace {{= }} with the value of the variable</td>
+        </tr>
+        <tr>
+            <td>Conditionals</td>
+            <td>
+                {{? expression }}<br />
+                    /* code here*/<br />
+                {{?}}<br />
+            </td>
+            <td>Render the code between the braces when the expression evaluates to true</td>
+        </tr>
+        <!-- <tr>
+            <td></td>
+            <td></td>
+            <td></td>
+        </tr> -->
+    </tbody>
+</table>
+
+### Flags
+
+No flags for Templating
+
+### Examples
+
+Open your terminal and go to the directory that you wish to use.
+
+    cd path/to/repo\ Folder
+
+Initialize Settings and Templates
+
+    react-component init -t
+
+This will create:
+
+    Repo Folder
+        | - .ccr
+            | - templates
+                | - component
+                |   | - component.dot
+                |
+                | - index
+                |   | - index.dot
+                |
+                | - style
+                |   | - style.css.dot
+                |
+                | - settings.json
+
+open `component/component.dot` and you should see something like this.
+
+        import React, { Component } from 'react';
+        {{? it.component.useCSS }}
+        import './{{= it.component.name }}.css';
+        {{?}}
+
+        class {{= it.component.name}} extends Component {
+          // constructor(props){
+            // super(props);
+            // this.state = {};
+          // }
+
+          // componentWillMount(){}
+          // componentDidMount(){}
+          // componentWillUnmount(){}
+
+          // componentWillReceiveProps(){}
+          // shouldComponentUpdate(){}
+          // componentWillUpdate(){}
+          // componentDidUpdate(){}
+
+          render() {
+            return (
+              <div></div>
+            );
+          }
+        }
+        export default {{= it.component.name}};
+
+> Note: All templates in the `.ccr/templates` folder are the default templates for create-components-react.
+
+`component/component.dot` is the default template to make each component file. You now can edit this file to make it look like anything you want as long as you follow the rules of javascript and doT. After you update this file and save it you can now use the `create` command and each component created will now take on the new look of the template file you updated.
```

Comment:
Can get rid of some of the 'now's, it's a given.

-> You can edit this file to make it look like anything you want as long as you follow the rules of javascript and doT. After you update this file and save it you can use the `create` command and each component created will take on the look of the template file you updated.